### DISCUSSION TITLE
Libreoffice too slow

### DISCUSSION THREAD

**Discussion by jquintinoscript** [microsoft/winget-pkgs#21831] [Q&A]
hi to all

the installation of the LibreOffice are too slow compare to the browers.

i user this 
winget install "ibreoffice"

use in cmd ou powershell to run 

I copy the link and use in the browser are too fast compare to the comand line. 
there is a way to resolve this ?
thanks 

jorge quintino

**Comment by OfficialEsco** [comment]
Check if adding
```
"network": {
       "downloader": "wininet"
   }
```
to your `winget settings` helps, the default setting uses Delivery Optimization.
